### DISCUSSION TITLE
Sprayer on iNav? cheap radar support from topradar.net

### DISCUSSION THREAD

**Discussion by DangerD1024** [iNavFlight/inav#7601] [Q&A]
Is there any way to add pump with water flowmeter to iNav to make crop sprayer like DJI Agras?

**Comment by DzikuVx** [comment]
If a pump can be activated with a standard servo protocol (PWM signal 50Hz, pulse length between 1000 and 2000us) then yes, it's possible

**Reply by DangerD1024** [reply]
Yes, it's PWM ESC, just need to add flowmeter, it's a hall sensor - sends pulses when after each 360* rotation.
